test_fzz.py

```diff
@@ -5,7 +5,12 @@
     fzz = pyfzz()
     data = fzz.read_file('sample_filt')
     bars = fzz.compute_zigzag(data)
+    print(f'Bars from file {bars}')
+    # Write the result into a file.
+    
```

Comment:
Hi Soham, can you please explain briefly to me what the input format and output format are? I am assuming list of python lists (or tuples) but just want to confirm. Also, do the bars contain dimension information also?

Reply:
Changed the README.